demo/summer-2016.html

```diff
@@ -240,23 +240,13 @@
     });
 
 
+    require(["ace/autocomplete/papeeria/tex_completer"], function(completer) {
```

Comment:
Any particular reason to use this path? I thought we discussed  lib/ace/ext/papeeria/autocompletion.coffee so that all Papeeria code was under the same folder.